{
  "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextDataList",
  "FileHeader" : {
    "MagicBytesLength" : 4,
    "MagicString" : "QTD\u0000",
    "VersionBytesLength" : 4,
    "VersionNumber" : 2
  },
  "FileSize" : 68,
  "QuestTextDataList" : [ {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "A Petty Thief's Woes",
    "MsgGmdIdx" : 0,
    "Type" : 0,
    "TypeName" : "QUEST_TEXT_TYPE_NAME"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "依頼者の願いを聞き入れ\r\n対象の敵を討伐する",
    "MsgGmdIdx" : 1,
    "Type" : 1,
    "TypeName" : "QUEST_TEXT_TYPE_ORDER"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "<NAME AREA>には\r\n誰かの救援を待っている者がいるようだ",
    "MsgGmdIdx" : 2,
    "Type" : 3,
    "TypeName" : "QUEST_TEXT_TYPE_FIND_INFO"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "準備を怠ったまま遠方に出向くやつはバカだ\r\n特に、ダンジョンの奥深くでのトラブルは\r\n死に直結する\r\n以前、何人かそんなやつらを助けたが、\r\n彼らは反省せず、同じ失敗を繰り返すだけだった\r\n学ばない者は、いくら素養があろうと大成せん",
    "MsgGmdIdx" : 3,
    "Type" : 4,
    "TypeName" : "QUEST_TEXT_TYPE_FIND_INFO_DETAIL"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "Defeat the enemy within the Catacombs",
    "MsgGmdIdx" : 4,
    "Type" : 2,
    "TypeName" : "QUEST_TEXT_TYPE_PURPOSE"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "Defeat the enemy encountered",
    "MsgGmdIdx" : 5,
    "Type" : 2,
    "TypeName" : "QUEST_TEXT_TYPE_PURPOSE"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "Report to the requester",
    "MsgGmdIdx" : 6,
    "Type" : 2,
    "TypeName" : "QUEST_TEXT_TYPE_PURPOSE"
  } ]
}